operational/flexera/cco/billing_centers_from_dimensions/billing_centers_from_dimensions.pt

```diff
@@ -0,0 +1,844 @@
+name "Flexera Billing Centers from Dimension Values"
+rs_pt_ver 20180301
+type "policy"
+short_description "Creates and manages billing centers structure based on existing dimension values. Please see the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/flexera/cco/billing_centers_from_dimensions/) for more information."
+long_description ""
+category "Operational"
+severity "low"
+default_frequency "15 minutes"
```

Comment:
It runs in small batches.  My experience was smaller batches of create/delete actions more frequently gave a better UX and optics that it was progressing since we don't have any visibility into the task label status of the CWF process running behind the scenes.